ui/src/pages/QuickSetup/ProvideServicesPage.tsx

```diff
@@ -293,22 +222,10 @@ const RequestForm = (props: {
   const hasAccountsForServices = requestInfo?.services?.reduce((acc, svc) => {
     if (existingServices.includes(svc)) return acc;
     const accounts = requestInfo?.accounts;
+
     switch (svc) {
       case ServiceKind.CLEARING:
```

Comment:
with 1 case, you probably don't need the switch anymore here

Reply:
Fixed